src/SignalR/clients/ts/signalr/src/HubConnection.ts

```diff
@@ -127,6 +127,22 @@ export class HubConnection {
         return this.connection ? (this.connection.connectionId || null) : null;
     }
 
+    get baseUrl(): string | null {
+        return this.connection ? (this.connection.baseUrl || null) : null;
```

Comment:
Can connection actually be null/undefined? Can baseUrl be?

Reply:
Connection shouldn't  be null. For baseUrl the check was there for non http scenarios but not really needed.

Reply:
We can at least get rid of the ternary expression.

Reply:
If we do eventually support non-http scenarios with the TS-client, we can just leave baseUrl as an empty string.